docs/PROCEDURES.md

```diff
@@ -0,0 +1,10 @@
+# Technical Steering Committee (TSC) Procedures
+## TSC Membership
+Membership changes to the L3AF TSC is governed by the [current list](https://github.com/l3af-project/governance/blob/main/L3AF_Committee_Members.md) of TSC members with voting rights. TSC membership changes require a majority vote by the current TSC members in order to pass. For more details on TSC voting procedures please refer to the [L3AF Technical Charter](https://github.com/l3af-project/governance/blob/main/docs/L3AF_technical_charter.md) "TSC VOTING" section.
+
+### Adding TSC Members
+In order to have a vote on adding a new member to the L3AF TSC a [github issue](https://github.com/l3af-project/governance/issues/new) should be opened against the governance repository. The title of the issue should be `TSC New Member - <NAME>` with a description on why this person should be added to the TSC. The opened github issue should then be referenced in the next TSC meeting which will determine a date to conduct a vote on the new membership. If a vote does not occur within a 1-month period from opening the issue the motion will be considered denied, and the issue will be closed.
+### Removal of TSC Members
+An existing TSC member can request their own removal from the L3AF TSC at anytime by opening a [github issue](https://github.com/l3af-project/governance/issues/new) with the title `TSC Membership Removal - <NAME>`. The issue will be taken up in the next weekly TSC meeting and the person will be removed as a member. If there is a proposed replacement for the member leaving, they should be mentioned in the issue. The proposed replacement member should open a new github issue and follow the [Adding TSC Members](#adding-tsc-members) section.
+
+Removal of a TSC member can be requested by other TSC members or from the community by opening a [github issue](https://github.com/l3af-project/governance/issues/new) with the title `TSC Membership Removal - <NAME>` with a description on why they should be removed. Examples of reasons for removal could be code of conduct violations or lack of activity of that TSC member for a prolonged period of time. Requests for removal will be discussed in the next weekly TSC meeting and a date will be determined for a vote on the membership removal.
```

Comment:
I would not recommend putting code of conduct violation details in a public issue.  
https://github.com/l3af-project/governance/blob/main/Code_of_Conduct.md says to send private email to Mike Dolan with the details.   So I think this line needs to be updated.

Reply:
I agree with David on this comment.  This should be kept separate and at most a reference to a link to the Code of Conduct file would be preference